src/peer_channels.rs

```diff
@@ -278,18 +278,24 @@ impl Context {
         })
     }
 
+    /// Returns [`None`] if the peer channel has not been created.
+    pub async fn get_peer_channel(&self) -> Option<tokio::sync::RwLockReadGuard<'_, Iroh>> {
```

Comment:
```suggestion
    /// Returns [`None`] if the peer channels has not been initialized.
    pub async fn get_peer_channels(&self) -> Option<tokio::sync::RwLockReadGuard<'_, Iroh>> {
```

Reply:
Overall it should be "peer_channels" everywhere, it's true that some naming inconsistency already exists here